### ISSUE TITLE
Multiple Cameras breaks DOMElement's position

### ISSUE THREAD

**Issue by futuremarc** [photonstorm/phaser#5128]
## Version

* Phaser Version: 3.23.0

* Operating system: MacOS 10.14.6

* Browser: All

## Description

It seems that adding more than 1 camera breaks the DOMElement GameObject. It's position seems to be offscreen to the right and it doesn't move at all.

## Example Test Code

```
 const div = document.createElement('div');
 div.innerText = 'DOM test';

const element = this.add.dom(500, 500, div, 'background-color: rgba(0, 255, 0, 0.2); width: 250px; height: 100px; font: 48px Arial; font-weight: bold');

const secondCamera = this.cameras.add(0, 0, 200, 200); // Breaks 'element'
secondCamera.ignore(element) // Does nothing
```

Anyone know anything about this? I can try to help pick this one up if anyone wants to pair

**Comment by mikewesthad** [comment]
I ran into this today. A DOM element is a single entity, so it's only going to have one final position on screen (or one "view"). When the camera manager renders the scene, it loops over the cameras in the order they were added. With each render, the DOM element's style gets updated to match the camera, so when the render finishes, it will match the last camera in the scene. If that last camera is set to ignore the DOM element, the DOM element gets "display: none" styling.

This doesn't seem like a bug to me, rather it seems like a limitation of DOM elements in the context of a multi-cam game. It'd be nice to have the ability to control which camera in the scene controls the DOM element rendering, but since Phaser supports parallel scenes already, you can just launch two scenes in parallel - one for the DOM elements and one for everything else.

**Comment by futuremarc** [comment]
Oh wow great answer, thank you! Do you have a good reference for launching multiple scenes at once?

**Comment by mikewesthad** [comment]
No problem. There are a few in [this folder](https://labs.phaser.io/index.html?dir=scenes/&q=) on phaser labs, e.g. [here](https://labs.phaser.io/edit.html?src=src\scenes\launch%20parallel%20scene.js) is how you can launch parallel scenes and [here](https://labs.phaser.io/edit.html?src=src\scenes\ui%20scene.js) is how you can run parallel scenes when the game boots.

**Comment by photonstorm** [comment]
I've updated the docs to inform devs about this, but I won't be changing how they work I'm afraid.
